packages/eslint-plugin-warp-drive/package.json

```diff
@@ -13,7 +13,8 @@
     "extends": "../../package.json"
   },
   "dependencies": {
-    "requireindex": "^1.2.0"
+    "requireindex": "^1.2.0",
+    "inflection": "3.0.2"
```

Comment:
this won't perfectly match warp-drive's inflection, but that's why its configurable in the rule